src/cml.js

```diff
@@ -378,20 +386,6 @@ class CML {
 
       if (url) return renderPr(url);
     } else {
-      await exec(`git config --local user.email "${userEmail}"`);
-      await exec(`git config --local user.name "${userName}"`);
-
-      if (CI) {
-        if (this.driver === GITLAB) {
-          const repo = new URL(this.repo);
-          repo.password = this.token;
-          repo.username = driver.userName;
-
-          await exec(`git remote rm ${remote}`);
-          await exec(`git remote add ${remote} "${repo.toString()}.git"`);
-        }
-      }
-
```

Comment:
Also would very much prefer `workflow rerun` rather than `rerun-workflow` as per conversation in https://github.com/iterative/cml/issues/827#issuecomment-985447709

Reply:
https://github.com/iterative/cml/blob/68e29bf0521079e447a56662852067f13b7d20db/src/cml.js#L344

Reply:
Awesome thanks didn't see that!